Sources/AirbnbSwiftFormatTool/swiftlint.yml

```diff
@@ -36,16 +36,23 @@ custom_rules:
     match_kinds:
     - identifier
     message: "Don't commit `print(…)`, `debugPrint(…)`, or `dump(…)` as they write to standard out in release. Either log to a dedicated logging system or silence this warning in debug-only scenarios explicitly using `// swiftlint:disable:next no_direct_standard_out_logs`"
-    severity: warning
+    severity: error
```

Comment:
We use `severity: error` for all of these in the apps repo so I updated that here (including for the new `no_unchecked_sendable`)